By default "wl authresp_mac_filter" and related iovar are disabled on wl1 and wl2
(BCM6755 radios). In result device replies (rejects) on auth requests instead
dropping them (desired behavior).

This patch enables WLAUTHRESP_MAC_FILTER macro which should enable support for
authresp_mac_filter both in wl and driver.
--- a/bcmdrivers/broadcom/net/wl/impl69/main/src/wl/config/wlconfig_lx_router_apsta
+++ b/bcmdrivers/broadcom/net/wl/impl69/main/src/wl/config/wlconfig_lx_router_apsta
@@ -116,3 +116,4 @@ WLC_OFFLOADS_M2M_INTR=1
 # STS_XFER_M2M_INTR=1
 HNDM2M=1
 BCM_CSIMON=1
+WLAUTHRESP_MAC_FILTER=1

